PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,119 @@
+# Quality Assurance Checklist
+To make reviews more efficient, please make sure the board meets the following standards and check everything off once the board meets the quality check. Once everything has been checked, the assigned reviewers will begin the review process. _Edit this description to check off the list._
+
+There are exceptions with all guidelines. As long as your decisions are justified, then you are good! Contact the reviewers or the leads about any exceptions.
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## Minimum Prerequisites
+- [ ] The board passes ERC and DRC.
+    - Note some ERC warnings are acceptable (**_CAUTION_** please be careful)
+    - Note some DRC warnings are acceptable (less so than ERC: **_CAUTION_** please be careful).
+- [ ] All traces are routed.
+- [ ] Units are in metric.
+- [ ] Gerbers are in a zip file in the main directory (after physical design is done)
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## Schematic Level Requirements 
+- [ ] Is proper noise resistance given to all peripheral devices (bypass caps and coils/ferrites)?  
+- [ ] Is proper ESD protection given to all MCU input pins (zener diodes)?  
+- [ ] Is proper power protection given to peripheral devices (zener diodes)?  
+- [ ] Are peripheral units used properly (reading datasheet)?  
+- [ ] Are testing points added at useful places?  
+- [ ] Is there proper short to GND protection at MCU outputs (inline resistors)?  
+- [ ] Do ADC inputs have caps?  
+- [ ] Are ADC inputs biased so there is room above expected value to determine if value is being overflowed?  
+- [ ] Are LED's located at useful places (comm, power, debugging, extra GPIO)?  
```

Comment:
LEDs*

Also perhaps we should check if the mechanical engineer or user (if any) is informed of the position of the LED and that it is in a highly visible or accommodative place (for the enclosure design).

e.g. putting the power LED behind two large electrolytics on the far side of the board and the board is all the way back at the end of the electronics box means that it's a lot more painful to debug on-site _or_
putting some controls LED at a very bad place on the PCB where the mechanical enclosure has to do a lot of gymnastics to make a hole for it that can be seen by the driver

Reply:
Checked for in section 3D Spacing.